sdk/keyvault/azure_security_keyvault_keys/Cargo.toml

```diff
@@ -14,9 +14,9 @@ keywords = ["sdk", "azure", "rest", "cloud", "keyvault", "security", "keys"]
 categories = ["api-bindings"]
 
 [dependencies]
-azure_core = { workspace = true }
+azure_core = { path = "../../core/azure_core" }
```

Comment:
This is all generated initially, and something @jhendrixMSFT will have to change in the emitter.

Reply:
It depends on how we want to treat azure_core.  I think we decided that we'll treat it as a workspace managed dependency, including whether packaged depend on the released or unreleased version.

In which case, all azure_core dependencies can be reverted back to `{ workspace = true }`

Reply:
I honestly can't remember. We've been back and forth so many times about which is better, this is why I asked for a write-up of what we discussed - what seems to have the best pro/con ratio considering our options.

Reply:
I've removed all toml edits from this PR.  we can address them in a new pr